ocaml/xapi/stream_vdi.ml

```diff
@@ -21,6 +21,7 @@ open Debug
 open Http
 open Forkhelpers
 open Pervasiveext
+open XXHash
```

Comment:
Avoid this. The interface should be small enough that using a qualified name is bearable.